### ISSUE TITLE
On message deleted (none returned values)

### ISSUE THREAD

**Issue by ImAzyx** [iBlitzkriegi/Vixio#286]
I made this issue to try to get answers about Vixio deleting messages.
When i try to send embed with deleted message author, message content, date, channel etc... the returned values are all none.

Is there any permissions needed ?
I tried to solve it by giving guild members intent and guild presences intent to bot but it's not the right way to solve it.

I'll let you my code:

on message delete seen by "{@botname}":

    create embed:
        set the title of embed to "%mention tag of author of event-message%"
        set the description of the embed to "Message : **%event-message%** by %mention tag of author of event-message% (%mention tag of event-channel%) was deleted"
    send the last created embed to channel with id "{@logs}" with event-bot

on skript load:
    create vixio bot:
        enable guild presences intent for bot
        enable guild members intent for bot
        login to "{@Token}" with the name "{@botname}"

**Comment by GTNTAnimations** [comment]
You have to set a variable to the contents of a message
EG:
**on guild message received seen by "{@Bot}":
    set {Text.%discord id of event-message%} to contents of event-message
    set {Channel.%discord id of event-message%} to "##%event-channel%"

on message deleted:
    create embed:
        set the title of embed to "Message deleted in %{Channel.%discord id of event-message%}%"
        set the description of embed to {Text.%discord id of event-message%}
        set the timestamp of embed to now
        set the color of embed to color from rgb 255, 87, 87 
    send the last created embed to channel with id "756593068789989414" with "{@Bot}"**

**Comment by ImAzyx** [comment]
Thanks for the quick answer ! I suppose that it work exactly for the on message edited event ;) !
And i wanted to know if its possible to share an event-member avatar url in author icon field please ?

**Comment by GTNTAnimations** [comment]
A: there isn't a "on message edited" event (kinda annoying)
B: wdym "if its possible to share an event-member avatar url in author icon field" Like put the icon of the user in the embed?

**Comment by ImAzyx** [comment]
yep that's it. Use the event-member avatar in an embed but not as thumbnail, as little  image (icon)
Well sorry i thought the on message edited event existed. So can we make an anti ghost ping with vixio ? I suppose yes, i'll find how to check if a player is mentioned in a message
